tests/test-cross-context-dispatch/ccd-common/src/main/java/org/eclipse/jetty/tests/ccd/common/Step.java

```diff
@@ -0,0 +1,18 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.tests.ccd.common;
+
+public interface Step
```

Comment:
The fashion these days appears to be to put any `abstract` classes and any impls in the same package inside of the `interface`.

Reply:
that seems noisy for no good reason.

Reply:
Take a look at the `Handler` class. I'm just getting in with this before Simone or Greg do :)